### PR TITLE
Ft api tests 156687751

### PR DESCRIPTION
**Pull request by dotNesh**
**What does this PR do?**
It has additional and fixed tests for the API

**Description of Task to be completed?**
Has completed additional and fixed tests for the following files:
`test_list_checks.py`
`test_admin.py`
`test_badge.py`
`test_pause.py`
`test_ensuretriggers.py`
`test_notify`
`test_check_model`
`test_sendalerts`
`test_create_check`
`test_ping`

**How should this be manually tested?**
cloning the repo and running `./manage.py test`

**What are the relevant pivotal tracker stories?**
#156687751

### CODE REVIEW

File: hc/api/tests/test_create_check.py [hunk 1]
```diff
@@ -47,23 +51,25 @@ def test_it_works(self):
...
+        self.assertEqual(r.status_code, 400)
+        self.assertEqual(r.json()['error'], "wrong api_key")
 
     def test_it_handles_invalid_json(self):
         ### Make the post request with invalid json data type
-        r = {'status_code': 400, 'error': "could not parse request body"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "could not parse request body")
+        r = self.client.post(self.URL,'data',content_type='application/json',HTTP_X_API_KEY='abc')
+        # r = {'status_code': 400, 'error': "could not parse request body"} ### This is just a placeholder variable
```
Comment (JoanNgatia) [inline hc/api/tests/test_create_check.py]: Please remove the commented out content

**Comment by JoanNgatia** [comment]
This looks good to me. Good job!
However, please remove the commented out code from the files.
I'll merge the branch to master once this is done.

**Comment by dotNesh** [comment]
Complete tasks on API tests

**Comment by JoanNgatia** [comment]
LGTM!

**Pull request merged**